test/no-reporter-test.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const metatests = require('../');
+const { spawn } = require('child_process');
+
+const test = metatests.test('must not fail when reporter is null');
+
+if (process.argv[2] === 'child-success') {
+  metatests.runner.instance.reporter = null;
```

Comment:
I don't like the fact, that the user of the library has to manually set `reporter` to `null` like that to disable reporting.
Can we maybe provide this as an option to `Runner` or add a method that will disable reporting?